src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -241,6 +241,10 @@ void ExecuteWithAbi (string [] supportedAbis, string apkInputPath, string apkOut
 								Log.LogDebugMessage ($"Skipping {path} as the archive file is up to date.");
 								continue;
 							}
+							if (apk.Archive.Any (e => e.FullName == path)) {
+								Log.LogMessage ("Warning: failed to add jar entry {0} from {1}: the same file already exists in the apk", name, Path.GetFileName (jarFile));
```

Comment:
Should this be a `LogDebugMessage` and not have the "warning" text in the message?

A developer probably can't (or won't) fix it if it is coming from a java library.

Reply:
good point